raitools/raitools/_managers/base_manager.py

```diff
@@ -56,7 +56,7 @@ def save(self, path):
 
     @staticmethod
     @abstractmethod
-    def load(path, rai_analyzer):
+    def _load(path, rai_analyzer):
```

Comment:
These methods are in private directory. do we need to make these private as well? Seems like unnecessary to me.

Reply:
yes, we are being extra careful, since other methods appear on managers and the methods on managers are public since they are exposed through the main RAIAnalyzer class, we've discussed this before